src/utils.ts

```diff
@@ -111,13 +111,14 @@ export function formatIpfsFile(ipfs: string | undefined): string | undefined {
 }
 
 export function formatHash(hash: string): string {
+  if (hash.length < 10) return hash;
```

Comment:
If the `hash` length is lesser than 10 digits it doesn't make much sense to shorten it..
It's not something that should happen, but the edge case that we pass in a already shortened hash would be taken into account here.